scss/components/filmstrip/_filmstrip.scss

```diff
@@ -11,6 +11,12 @@
 
 // Stanford Earth styling for the filmstrip top-border
 .filmstrip {
+  .front & {
```

Comment:
It is very possible that the front page have more than one filmstrip. Infact, I know it does. Please revise this to a front page specific stylesheet that targets the specific block you are trying to pull up over another element. This is far too general.